receiver/namedpipereceiver/metadata.yaml

```diff
@@ -0,0 +1,9 @@
+type: namedpipe
+
+status:
+  class: receiver
+  stability:
+    alpha: [logs]
+  distributions: [contrib]
```

Comment:
this is not in the contrib distribution yet.

Reply:
I can be added once this PR is merged